lib/builders/encounter-query.js

```diff
@@ -3,6 +3,10 @@
 const buildReferenceFilter = require('./reference-filter')
 const resourceQueryBuilder = require('./resource-query')
 
+function buildClassFilter (encounterClass) {
+  return { class: encounterClass }
```

Comment:
@tumbledwyer Please stick to the convention in the rest of the code of not using bracket spacing e.g. https://github.com/bausmeier/fhir-store/blob/8ec1a32cb6cb49f04b9c0f46a6d26d0bd7415408/lib/builders/practitioner-query.js#L21